vertinimai.md

```diff
@@ -63,22 +63,3 @@
 | i | 2 | 2 | 2 | 2 |
 | d | 3 | 2 | 4 | 3 |
 11.23
-
```

Comment:
Don't delete this!! I had just fixed it to have the latest marks!!

![monkas](https://user-images.githubusercontent.com/68472824/208489672-35fd1992-f416-4001-a596-6bb6be0e3bf4.png)